src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -69,7 +72,8 @@ public static void deleteAllTokens(User user) {
     httpDelete.setHeader(X_WIKIA_INTERNAL_REQUEST, "0");
 
     try (CloseableHttpResponse response = getDefaultClient().execute(httpDelete)) {
-      PageObjectLogging.log("DELETE HEADERS: ", response.toString(), true);
+      PageObjectLogging.log("DELETE TOKENS REQUEST: ", httpDelete.toString(), true);
+      PageObjectLogging.log("DELETE TOKENS RESPONSE: ", response.toString(), true);
     } catch (ClientProtocolException e) {
       PageObjectLogging.log("CLIENT PROTOCOL EXCEPTION", ExceptionUtils.getStackTrace(e), false);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "CLIENT PROTOCOL EXCEPTION" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)